### ISSUE TITLE
Child handler doesn't stopPropagation since 1.1.4 release

### ISSUE THREAD

**Issue by Colmea** [greena13/react-hotkeys#106]
Hello,

First thank you for your package.

Since the 1.1.4 release, and especially this Pull Request (https://github.com/greena13/react-hotkeys/pull/101/files), there is a breaking change regarding event propagation to parent's handler.

Before this release, a hotkey defined in a child component AND in a root component was only triggered once (by the most nested component). But now, they are always both triggered (nested component AND root component).

I guess this happens because we create a new handler on each render() (and so `hasChanged(handlers, prevHandlers)` is always true) (see: https://github.com/greena13/react-hotkeys/pull/101/files#diff-5f147d6760a1bc2314f409768a38984fR247).

It means that your example in the README won't stop propagation to parent because a new handler is created on each render:
```
const MyNode = React.createClass({
  render() {
    const handlers = {
      'deleteNode': this.deleteNode
    };

    return (
      <HotKeys handlers={handlers}>
        Node contents
      </HotKeys>
    );
  }
});
```

Is it a (new) expected behavior ? If its is, shouldn't we mark this PR as a breaking change ?

We had to rollback to previous release to fix this issue, and I guess we're not the only one to create a new handler on each render (bad idea though, but a breaking change nevertheless)

**Comment by greena13** [comment]
Hello Colmea, thank you for your patience on this.

Can you please confirm the latest version of the package that still works for you? The `hasChanged()` logic appears to have been in since `v1.1.1`.

**Comment by natew** [comment]
I'm seeing this as well, anyone have a workaround? I can submit a PR, is this still maintained? I love the API but afraid this has been open so long it's a dead project.

**Comment by Colmea** [comment]
Hey @greena13 , sorry for the delay.
Indeed, the  `1.0.6` is the last working release for us.

**Comment by greena13** [comment]
This will be resolved in [v2.0.0-pre1](https://github.com/greena13/react-hotkeys/releases/tag/v2.0.0-pre1) when it's released soon.
